src/screens/ContactScreen/index.js

```diff
@@ -250,7 +250,7 @@ const ContactScreen = ({navigation}) => {
 const styles = StyleSheet.create({
   container: {
     display: 'flex',
-    backgroundColor: '#FFFFFF',
+    backgroundColor: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded color value to a reference from the `COLORS` object is a good practice for maintainability and consistency. This allows for easier updates in the future if the color scheme changes.
```diff
-    backgroundColor: '#FFFFFF',
+    backgroundColor: COLORS.white,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->